homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
```

Comment:
`ConfigFlowHandler` -> `HuaweiLTEConfigFlow`

Reply:
Noted in a088ef9e3596912948152b313def512c99545a40

Reply:
https://github.com/home-assistant/core/pull/147910